### DISCUSSION TITLE
[TASK] Will the degraded state affect normal use?

### DISCUSSION THREAD

**Discussion by w7team** [longhorn/longhorn#9154] [General]
If a volume is set to have 3 replicas, but I currently only have 2 nodes, the volume is in a degraded state. Is there any impact on usage? It seems like I can still use it normally.

如果给卷设置了3个副本，但是我目前只有2个节点。此时卷处于降级的状态。使用起来有没有什么影响？我看也能正常使用

**Comment by ChanYiLin** [comment]
It should work without any problem, but not recommend.
The reasons are mostly from operation perspective.
The controller will still reconcile unassigned replicas, might introduce some cpu burden if there are lots of them.
Besides, if there are many volumes that are in degraded state because of unassigned replicas, you won't be able to quickly find the volumes having failed replicas.
Moreover, there is a risk that the cluster is running out of the spaces but you won’t be made aware immediately.


We recommend set the setting [Allow Volume Creation with Degraded Availability](https://longhorn.io/docs/1.6.2/best-practices/#allow-volume-creation-with-degraded-availability) to `false`
